### PR TITLE
50086_display_all_requests

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

**Comment by shakil-fg** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
OK

File: app/views/requests/_requests.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<tr>
+  <% @requests.each do |request| %>
+    <td><%= request.title %></td>
+    <td><%= request.author %></td>
+    <td><%= request.publisher %></td>
+    <td><%= request.link %></td>
+    <td><%= request.status %></td>
+    <td>
+      <%= link_to t(".cancel_button"), request_path(request), class: 
+          "btn btn-danger", method: :delete, data: {confirm: t(".you_sure")} %>
```
Comment (MuhammadTamzid) [inline app/views/requests/_requests.html.erb]: not delete. just update the status :)

File: config/routes.rb [hunk 2]
```diff
@@ -13,6 +12,6 @@
     end
 
     devise_for :users, controllers: {registrations: "registrations"}
-    resources :requests, only: [:new, :create]
+    resources :requests, only: [:index, :new, :create, :destroy]
```
Comment (MuhammadTamzid) [inline config/routes.rb]: user can not destroy request

File: app/controllers/requests_controller.rb [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
+      flash[:success] = t ".flash_request_success"
       redirect_to root_path
     else
-      flash[:success] = I18n.t ".flash_request_success"
+      flash[:success] = t ".flash_request_success"
       render :new
     end
   end
 
+  def destroy
```
Comment (MuhammadTamzid) [inline app/controllers/requests_controller.rb]: remove

**Comment by MuhammadTamzid** [comment]
Ok

**Pull request merged**